packages/main/src/handlers/index.ts

```diff
@@ -67,10 +68,11 @@ export const registerHandlers = () => {
   ipcMain.on('scrape', async (event: IpcMainEvent) => {
     const config = await getConfig();
     const eventSubscriber = new BudgetTrackingEventEmitter();
-    scrapeAndUpdateOutputVendors(config, eventSubscriber);
```

Comment:
@baruchiro  we had a bug here, listening to events was set up only after running scrape, and scraping was without await, so there was a race condition on whether or not the first event set would be caught by the onAny handler or not.